hw2/05-form.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>05 - Form</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
+    <script defer src="05-form.js"></script>
+  </head>
+  <body class="bg-dark p-5">
+    <main>
+      <div class="d-flex align-items-center justify-content-center">
+        <form class="d-inline-block p-4 bg-white h-60%" id="form1">
+          <h2 class="text-center">Exercise 05 FORM</h2>
+          <div class="form-group row">
+            <div class="col-form-label">
+              <label for="Username" class="fw-bold">Full Name:</label>
+              <input type="text" class="form-control" id="Username" required/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <div class="col-form-label">
+              <label for="email" class="fw-bold">Email:</label>
+              <input type="email" class="form-control" id="email" required/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <div class="col-form-label">
+              <label for="registrationStatus" class="col-form-label fw-bold"
+                >Registration Status:</label
+              >
+              <select
+                class="form-select"
+                name="reg-status"
+                id="registrationStatus"
+              >
+                <option value="">Choose an option</option>
+                <option value="registered">Registered</option>
+                <option value="non-registered">Not Registered</option>
+                <option value="waitlist">Waitlist</option>
+              </select>
+            </div>
+          </div>
+          <fieldset class="form-elemnet form-checkboxes">
+            <legend class="col-form-label fw-bold">
+              Have you taken any of following courses?
+            </legend>
+            <br />
```

Comment:
Avoid using the <br> tag to add spacing between different html elements. Use the <br> tag to enter line breaks, not to add space between form elements. The <br> element has a single, well-defined purpose — to create a line break in a block of text. To add spacing between different html elements use CSS ( for example the margin property) instead.